app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -169,8 +171,14 @@ export class TitusRunJobStageConfig extends React.Component<IStageConfigProps, I
     this.forceUpdate();
   };
 
+  public disruptionBudgetChanged = (values: IJobDisruptionBudget) => {
+    const { stage, stageFieldUpdated } = this.props;
+    stage.cluster.disruptionBudget = values;
+    stageFieldUpdated();
+  };
```

Comment:
This is only needed to sync updates between the parent angular controller and the child react component.